web/src/app/core/services/sorting.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+
+/**
+ * Sorting service
+ */
+@Injectable({
+  providedIn: 'root',
+})
+export class SortingService {
+  /**
+   * This function is used for sorting the list by date field
+   */
+  sortList(list: any, field: any): any {
```

Comment:
I think using a generic here might be better.

Also I just realised it is not returning anything but has a return  type

Reply:
I will change the return type to void. As this function is not returning anything. Just modifying the list

Reply:
Pushed the code with the fix.